compute/metadata/go.mod

```diff
@@ -1,3 +1,5 @@
 module cloud.google.com/go/compute/metadata
 
 go 1.20
+
+require golang.org/x/sys v0.22.0
```

Comment:
the first dependency 🥲

Reply:
sad

Reply:
this module specifically is a dependency magnet (xref https://github.com/googleapis/google-cloud-go/issues/6311 https://github.com/kubernetes/kubernetes/issues/113366)

x/sys isn't the end of the world as a transitive dependency, but is there anything guarding against pulling in cloud dependencies for this?

Reply:
No. We will have a PR landing in the next month that will add one more dep to this package though that is a "cloud dep". But that should not pull in any more explicit deps. See https://github.com/googleapis/google-cloud-go/pull/11078